### PR TITLE
Capd env

### PR DESCRIPTION
**Pull request by alexander-hughes**

### CODE REVIEW

**Comment by alexander-hughes** [comment]
@jimmytipit  having some difficulty getting the rendered code blocks to display correctly.  Hoping to see them formatted as solid blocks instead of individual code lines.  Can you take a look and provide some guidance please?

File: src/pages/blog/cluster-api-development-environment.md [hunk 1]
```diff
@@ -0,0 +1,197 @@
...
+Kubernetes-style APIs to cluster creation, configuration, and management. It provides optional, additive functionality
+on top of core Kubernetes to manage the lifecycle of a Kubernetes cluster.
+
+In a previous [blog post](
+https://www.airshipit.org/blog/airship-blog-series-5-drydock-and-its-relationship-to-cluster-api/), Alan Meadows and
+Rodolfo Pacheco discussed the evolution of Airship 1.0 to Airship 2.0 and the relationship between [Drydock](
+https://opendev.org/airship/drydock) and [Cluster API](https://cluster-api.sigs.k8s.io/). It's an interesting read,
+looking at how Cluster API will be used by Airship 2.0.
+
+Today I will provide you the documentation and my tested step-by-step directions to creating a Cluster API development
```
Comment (drewwalters96) [inline src/pages/blog/cluster-api-development-environment.md:33]: "to creating"

File: src/pages/blog/cluster-api-development-environment.md [hunk 2]
```diff
@@ -0,0 +1,197 @@
...
+
+11. Interact with your cluster
+
+    ```shell script
+    kubectl --kubeconfig=./work-cluster.kubeconfig get nodes
+    ```
+
+<br>
+
+That's all there is to it! If you made it this far you should have a working CAPD environment to develop in.
```
Comment (drewwalters96) [inline src/pages/blog/cluster-api-development-environment.md:193]: comma after "If you have made it this far"

File: src/pages/blog/cluster-api-development-environment.md (hunk 1 above)
Comment (alexander-hughes) [reply src/pages/blog/cluster-api-development-environment.md:33]: not clear on your comment, is this for line 33?  Current wording is "Today I will provide you the documentation and my tested step-by-step directions to creating a Cluster API development"

File: src/pages/blog/cluster-api-development-environment.md (hunk 2 above)
Comment (alexander-hughes) [reply src/pages/blog/cluster-api-development-environment.md]: done

**Comment by alexander-hughes** [comment]
> 
> 
> This is nice. A couple of grammar items in-line.
> 
> Do you have any examples of exercises you did to test your environment? I think it would be good to share those, especially if they relate to how Airship will use Cluster-API. Do we have any updates we can share about our usage of the Cluster-API implementation?

So far this is just getting the environment working, and the blog post is a request from the CAPI community as instructions on setting up the environment until official documentation is updated and made more clear.

**Comment by gcutrini** [comment]
> @jimmytipit having some difficulty getting the rendered code blocks to display correctly. Hoping to see them formatted as solid blocks instead of individual code lines. Can you take a look and provide some guidance please?

This has been addressed in #89

**Comment by alexander-hughes** [comment]
> 
> 
> > @jimmytipit having some difficulty getting the rendered code blocks to display correctly. Hoping to see them formatted as solid blocks instead of individual code lines. Can you take a look and provide some guidance please?
> 
> This has been addressed in #89

Thanks!  Seems to be working correctly now!

**Comment by alexander-hughes** [comment]
> 
> 
> looks nice to me, thanks for the shoutout =)

Great to hear, thanks for looking it over

**Comment by mattmceuen** [comment]
Hey this is great, Alex.  Just a couple of suggestions:

1) Maybe mention `kind` up toward the top somewhere, e.g. "Today I will provide you the documentation and my tested step-by-step directions to creating a Cluster API development environment using kind", or even in the title of the post.  I think this will help set expectations on "how deep a stack am I setting up"; when I hear kind I think "easy" personally :)

2) In your intro paragraph, maybe add a small blurb about what you can do with the dev env -- e.g. deploy virtual nodes (as docker containers in kind), test out changes to the Cluster API codebase, etc.  Just to set the context of "why would I want to install this thing".

**Pull request merged**